deploy/crds/hwameistor.io_localdiskclaims_crd.yaml

```diff
@@ -40,6 +40,11 @@ spec:
           spec:
             description: LocalDiskClaimSpec defines the desired state of LocalDiskClaim
             properties:
+              consumed:
+                description: Consumed represents disks backing the claim has already
+                  consumed by consumer. If true the claim will be deleted soon form
+                  kubernetes
```

Comment:
```suggestion
                description: Consumed represents disks backing the claim has already
                  been consumed by a consumer. If true the claim will be deleted soon
                  from kubernetes
```

Reply:
Ok, this file was generated by operator-manager, i will update descriptions in go files and update this crd file again.